cmd/complete.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/mmerkes/clerk/pkg/storage"
+	"github.com/spf13/cobra"
+)
+
+// completeCmd represents the complete command
+var completeCmd = &cobra.Command{
+	Use:   "complete",
+	Short: "Complete a task permanently",
+	Long:  `Complete a task permanently and set the EndTime.`,
```

Comment:
`permanently` unnecessary here because it implies many things, which might not be true if we allow you to reopen a task. However, fine for now.